content/events/2025-07-12-rock-camp-lisle/index.md

```diff
@@ -0,0 +1,13 @@
++++
+address = ''
+artists = ['theironspiders']
+city = 'Lisle'
+date = 2025-07-12
```

Comment:
this is the date of the post itself, not of the event. please update this to today.